roles/splunk_common/tasks/get_facts.yml

```diff
@@ -40,7 +40,7 @@
   set_fact:
     splunk_target_version: "{{ splunk.build_location | regex_search(regexp, '\\1') | default('0') }}"
   vars:
-    regexp: 'splunk(?:forwarder|light|cloud)?-((\d+)\.(\d+)\.(\d+))-'
+    regexp: 'splunk(?:forwarder|light|cloud|-unstripped)?-((\d+)\.(\d+)\.(\d+))'
```

Comment:
You only care about the last digits with this, right?  (i.e. 7.2.5)
So why not just use this ```splunk\D*?-((\d+)\.(\d+)\.(\d+))$```